### ISSUE TITLE
Identify target devices / browsers that should be supported for viewing cdevents.dev site

### ISSUE THREAD

**Issue by tdcox** [cdevents/cdevents.dev#16]
Consensus is needed on the expected target devices which should be compatible with cdevents.dev

Of particular interest is what expectations are about rendering the spec in a readable form on small devices like phones?

**Comment by e-backmark-ericsson** [comment]
Given that one of the main purposes of the site should be to be the storefront for CDEvents, I believe we should strive for a good rendering of the site also on phones. Especially the main page(s). The detailed spec pages are not as important to render as nicely, since they might be more frequently accessed from a laptop browser than from a mobile phone.

**Comment by tdcox** [comment]
Docsy has some issues on smaller screens, so that may mean we have to do some work on the templating to get it working consistently. The spec is currently readable on a phone, but there is a lot of sideways scrolling needed to navigate the tables.

**Comment by e-backmark-ericsson** [comment]
> We can turn tables in a different format if that is more portable.

I agree. I'm open for other formats on those tables if that would improve the situation, rather than doing a lot of templating work.
With the differences you describe there @tdcox , it sounds like we'd need to review all docs changes both on a laptop and a phone before merging them. At least if they affect any formatting. I guess that should be documented somewhere in a contributor guide or similar.

**Comment by tdcox** [comment]
I would suggest that docsy’s current rendering of tables is actually the preferred solution because it keeps the text at a readable size, rather than rescaling the table to fit on the screen, requiring both zooming and scrolling in order to read.

The limitation is in how many words you can fit across a small display at a readable size. We have to be mindful of accessibility here. Too many websites are inaccessible to the over 25s because they use small fonts to gain screen real estate and older people’s eyesight is no longer good enough to resolve them.

I would say that we are close to being at the best available compromise for phone viewing already, unless the whole site was converted to narrative form, which would probably make the desktop experience worse. There are some small tweaks that we can make to the tables to improve rendering but sideways scrolling is always going to be a limitation on phones.
